lib/cache/IdleFileCachePlugin.js

```
@@ -181,6 +181,20 @@ class IdleFileCachePlugin {
 				isInitialStore = false;
 			}
 		};
+
+		// Safety flush to drain all remaining pending tasks and persist the pack
+		const onBeforeExit = () => {
+			if (!pendingIdleTasks.size) return;
+			currentIdlePromise = Promise.all([
+				...[...pendingIdleTasks.values()].map(fn => Promise.resolve().then(fn)),
```

ESLint

[error] Replace `fn·=>·Promise.resolve().then(fn)` with `(fn)·=>⏎↹↹↹↹↹Promise.resolve().then(fn)⏎↹↹↹↹`

(prettier/prettier)